src/tests/multihost/alltests/test_automount_from_bash.py

```diff
@@ -0,0 +1,1000 @@
+""" Automation of Auto Mount suite
+:requirement: Ldap Provider - automount
+:casecomponent: sssd
+:subsystemteam: sst_idm_sssd
+:upstream: yes
+:status: approved
+"""
+import time
+import pytest
+import os
+import ldap
+import subprocess
+from sssd.testlib.common.utils import sssdTools, LdapOperations
+from constants import ds_instance_name, ds_suffix, ds_rootdn
+from sssd.testlib.common.paths import SSSD_DEFAULT_CONF
+
+
+def find_logs(multihost, log_name, string_name):
+    """This function will find strings in a log file
+    log_name: Absolute path of log where the search will happen.
+    string_name: String to search in the log file.
+    """
+    log_str = multihost.client[0].get_file_contents(log_name).decode('utf-8')
+    assert string_name in log_str
+
+
+def clear_only_domain_log(multihost):
```

Comment:
this 'delete_sssd_domain_log' is available in utils.py

Reply:
This function is not going to work here .  Here i am not removing any logs rather i am emptying the domain log.
Test fails if i use the recommended function